app/src/main/java/com/cornellappdev/scoop/ui/components/post/FirstPage.kt

```diff
@@ -0,0 +1,196 @@
+                .padding(top = 240.dp)
+        ) {
+            TransportationSection(
+                typeText,
+                setTypeText,
+                modifier = Modifier.onFocusChanged {
+                    rideState.value.type = typeText
+                }
+            )
+            Spacer(
+                modifier = Modifier.height(24.dp)
+            )
+            CityPicker(
+                cityState = departureText,
+                placeholder = "Departure location",
+                icon = Icons.Filled.NearMe,
+                modifier = Modifier.onFocusChanged {
+                    rideState.value.departureLocation = departureText.value
+                },
+                disableDivider = true,
+                placeholderColor = Color(0xFF001E2D),
+            )
+            Spacer(
+                modifier = Modifier.height(24.dp)
+            )
+            CityPicker(
+                cityState = arrivalText,
+                placeholder = "Arrival location",
+                icon = Icons.Filled.Place,
+                modifier = Modifier.onFocusChanged {
+                    rideState.value.arrivalLocation = arrivalText.value
+                },
+                disableDivider = true,
+                placeholderColor = Color(0xFF001E2D),
+            )
+
+            Column(
+                modifier = Modifier
+                    .align(Alignment.End)
+                    .wrapContentSize()
+                    .fillMaxWidth()
+                    .fillMaxHeight(),
+                verticalArrangement = Arrangement.Bottom
+            ) {
+                Button(
+                    modifier = Modifier
+                        .align(Alignment.End)
+                        .padding(bottom = 118.dp)
+                        .width(86.dp)
+                        .height(51.dp),
+                    shape = RoundedCornerShape(26.dp),
+                    enabled = proceedEnabled,
+                    onClick = {
+                        when (typeText) {
+                            null -> {
+                                showInvalidTypeMessage = true
+                                proceedEnabled = false
+                                coroutineScope.launch {
+                                    disableMessage()
+                                }
+                            }
+                            else -> {
+                                // Updates trip state with details collected on FirstPage
+                                val ride = rideState.value
+                                ride.type = typeText
+                                rideState.value = ride
+                                onProceedClicked()
+                            }
+                        }
+                    },
+                    contentPadding = PaddingValues(10.dp),
+                    colors = ButtonDefaults.buttonColors(
+                        backgroundColor = Color(0xFFCEE9DC),
```

Comment:
is there no materialTheme for this style? If there is you should add it for future buttons